website/content/docs/discovery/dns.mdx

```diff
@@ -168,15 +168,24 @@ foobar.node.dc1.consul.	0	IN	A	10.1.10.12
 
 ### RFC 2782 Lookup
 
-The format for RFC 2782 SRV lookups is:
+Valid formats for RFC 2782 SRV lookups are:
 
-    _<service>._<protocol>[.service][.datacenter][.domain]
+```shell-session
+# 1: No filtering on tag
```

Comment:
I need to make these lines show up as comments. Right now, they look like regular content/text/code.

Reply:
Changed format